docs/includes/quick-start/troubleshoot.rst

```diff
@@ -0,0 +1,7 @@
+.. note::
+
+   If you run into issues on this step, ask for help in the
```

Comment:
S: since this note is included on tutorial pages with multiple steps, I think this should say "these steps" rather than "this step"
```suggestion
   If you run into issues on these steps, ask for help in the
```

Reply:
unless each page of the quick start is a step, and the steps on each individual page are sub-steps?

Reply:
Good point. I'll omit "on this step" for this statement across all the pages to eliminate any confusion.